Src/IronPython/Runtime/Types/DocBuilder.cs

```diff
@@ -599,6 +602,17 @@ static class DocBuilder {
                     xpd = _CachedDoc;
                 } else {
                     xpd = new XPathDocument(xml);
+                    if (isRef) {
+                        // attempt to redirect if required
+                        var node = xpd.CreateNavigator().SelectSingleNode("/doc/@redirect");
+                        if (node != null) {
+                            var redirect = node.Value;
+                            redirect = redirect.Replace("%PROGRAMFILESDIR%", Environment.GetFolderPath(Environment.Is64BitProcess ? Environment.SpecialFolder.ProgramFilesX86 : Environment.SpecialFolder.ProgramFiles) + Path.DirectorySeparatorChar);
```

Comment:
Can you use Environment.ExpandEnvironmentVariables? https://msdn.microsoft.com/en-us/library/system.environment.expandenvironmentvariables%28v=vs.110%29.aspx?f=255&MSPPError=-2147217396

Not sure if it takes into account the 64-bit vs. 32-bit

Reply:
Unfortunately not because it's not actually an environment variable...